anyway/parsers/road_segments.py

```diff
@@ -38,6 +38,7 @@ def _iter_rows(filename):
 
 
 def parse(filename):
+    RoadSegments.query.delete()
     for batch in batch_iterator(_iter_rows(filename), batch_size=50):
         db.session.bulk_insert_mappings(RoadSegments, batch)
         db.session.commit()
```

Comment:
Does this mean we do a commit for each 50 row?

Reply:
Yes, I assume we can increase it. However it's a very small table (~1000 rows) - that is loaded once - no real significance here.